sqlmesh/core/console.py

```diff
@@ -270,8 +270,13 @@ def log_error(self, message: str) -> None:
         """Display error info to the user."""
 
     @abc.abstractmethod
-    def log_warning(self, message: str) -> None:
-        """Display warning info to the user."""
+    def log_warning(self, message: str, logger_message: t.Optional[str] = None) -> None:
```

Comment:
The fact that the console uses a logger is an implementation detail and shouldn't leak into the interface. I suggest calling this `short_message` and `long_message` instead. The long_message may have applications in future other than just printing into the log file